src/vdaf/prg.rs

```diff
@@ -137,15 +135,11 @@ impl Prg<16> for PrgAes128 {
 }
 
 /// The key stream produced by AES128 in CTR-mode.
-#[derive(Debug)]
-pub struct SeedStreamAes128(Aes128Ctr);
+pub struct SeedStreamAes128(Ctr64BE<Aes128>);
 
 impl SeedStreamAes128 {
     pub(crate) fn new(key: &[u8], iv: &[u8]) -> Self {
-        SeedStreamAes128(Aes128Ctr::from_block_cipher(
-            Aes128::new(GenericArray::from_slice(key)),
-            GenericArray::from_slice(iv),
-        ))
+        SeedStreamAes128(Ctr64BE::<Aes128>::new(key.into(), iv.into()))
```

Comment:
Yes, `Ctr64BE` is correct, this is what `Aes128Ctr` used internally as of `aes 0.7.5` and `ctr 0.8.0` (which `aes` optionally pulled in before).